client/src/components/TagSelector/TagSelector.style.ts

```diff
@@ -94,15 +95,214 @@ export const Title = styled.h3`
 	border: 2px solid #777;
 `;
 
-export const Filter = styled.input`
+const Filter = styled.input`
 	display: flex;
 	padding: 0.2rem 0.4rem;
 	border-radius: 3px;
-	border: 2px solid #ccc;
+	outline: 2px solid #ccc;
 	box-shadow: 0.1rem 0.1rem 0 0 #ddd;
-	margin-top: -1rem;
+	border: none;
 	align-self: flex-end;
 	max-width: 150px;
 
 	font-size: 0.88rem;
+
+	&:focus {
+		outline-color: dodgerblue;
+		box-shadow: 0rem 0.5rem 0.2rem -0.2rem #ccc;
+	}
+`;
+
+const ClearFilter = styled.button`
```

Comment:
Rename `ClearFilter`->`FilterClearButton`.

Reply:
I also don't like the styling of this, but that's probably not important enough to worry about right now.